### ISSUE TITLE
add focus shifting when activating contextual notes

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/startwords#169]
the main concern from John Jameson's accessibility review is the accessibility of contextual notes for SRs/keyboard users:

> This tooltip is suuuuuper janky on keyboards and doesn’t work at all on screen readers, because the actual content is somewhere else on the page. If it’s going to be inline, make it inline in the DOM too (so the text appears in context for assistive devices — Tippy is great for this ). If it needs to be at the end of the DOM, take over and trap the cursor in a modal dialog.  I can be more verbose on this if you want :-D 

we're the latter case (the content needs to be at the end of the DOM, since that's where it gets rendered as an endnote), so we need to implement some automatic focus shifting so that the content of contextual notes is automatically focused when the note is activated.

JJ hints at the contextual note behaving like a modal, which is interesting - maybe we will also need to apply/remove some ARIA so that the SR user knows that when the note is active, they need to close it to continue reading.
